src/api/worker/facades/lazy/MailFacade.ts

```diff
@@ -849,3 +852,21 @@ function recipientToEncryptedMailAddress(recipient: PartialRecipient): Encrypted
 		address: recipient.address,
 	})
 }
+
+/**
+ * Verify all attachments contain correctly formatted MIME types. This ensures that they can be sent.
+ *
+ * Note that this does not verify that the mime type actually corresponds to a known MIME type.
+ * @param attachments
+ * @throws {Error} if a MIME type is not correctly formatted for at least one attachment
+ */
+export function validateMimeTypesForAttachments(attachments: Attachments) {
+	const regex = /^\w+\/[\w.+-]+$/g
+	for (const attachment of attachments) {
+		if (isDataFile(attachment) || isFileReference(attachment)) {
+			if (!attachment.mimeType.match(regex)) {
+				throw new Error(`${attachment.mimeType} is not a correctly formatted mimetype (${attachment.name})`)
```

Comment:
this should either be a ProgrammingError or a UserError depending on who controls that value

Reply:
I think ProgrammingError makes sense as the user isn't directly responsible for the mime types (it's the client's).

Reply:
Basically, if we hit this error, there is a bug somewhere that needs fixed.